packages/terra-progress-bar/src/ProgressBar.jsx

```diff
@@ -23,21 +23,29 @@ const propTypes = {
    * Sets the maximum possible fill-value.
    */
   max: PropTypes.number,
+  /**
+   * Boolean check for gradient.
+   */
+  hasGradient: PropTypes.bool,
 };
 
 const defaultProps = {
   heightSize: 'small',
   max: 100,
   value: 0,
+  hasGradient: false,
 };
 
 const ProgressBar = ({
   heightSize,
   value,
   max,
+  hasGradient,
   ...customProps
 }) => {
   const classes = cx([
+    // Add this class if gradient is true
```

Comment:
You can probably remove this comment, I added it within the sample to help describe what it was doing, but in the context of classnames this syntax is used all over and is self documenting

Reply:
My bad. removed comment. Thanks @StephenEsser